requirements-dev.txt

```diff
@@ -1,15 +1,15 @@
 -e .
-aioconsole==0.1.15
```

Comment:
I don't think that pin changes are needed for integrating setuptools-scm but could be useful separately...

Reply:
I don't think that changing the support matrix needs to be coupled with the packaging approach refactoring really.